kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -1,11 +1,20 @@
-{% extends "kolibri/base.html" %}
```

Comment:
I would prefer not to go the latter route - allowing arbitrary data to be bootstrapped into the template depending on exactly which Django view it is instantiated in sounds like it would make the template overly complex/verbose.